components/menu/calendar/Screens/DashboardScreen/index.js

```diff
@@ -210,6 +211,7 @@ export default class DashboardScreen extends Component {
          return;
        }
        this.props.navigation.navigate('HomeScreen', { description });
+       // eslint-disable-next-line consistent-return
```

Comment:
this is a valid linter error. the first if statement should return an empty string instead of returning void

Reply:
I update to return empty string. @t98b @mohhef please verify that the functionality of the function remains the same after this change.